src/test/java/apicurito/tests/steps/VerificationSteps/AssertAll.java

```diff
@@ -0,0 +1,19 @@
+package apicurito.tests.steps.VerificationSteps;
```

Comment:
package names should be lowercase. Also no reason to call it `steps.verificationsteps`, since `steps.verification` is clear enough

Reply:
updated